bin/sozo/src/commands/execute.rs

```diff
@@ -17,27 +20,64 @@ use super::options::transaction::TransactionOptions;
 use super::options::world::WorldOptions;
 use crate::utils;
 
+#[derive(Debug, Clone)]
+pub struct CallArguments {
+    pub tag_or_address: ResourceDescriptor,
+    pub entrypoint: String,
+    pub calldata: Vec<Felt>,
+}
+
+impl FromStr for CallArguments {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self> {
+        let parts = s.splitn(3, ",").collect::<Vec<_>>();
+
+        if parts.len() < 2 {
+            return Err(anyhow!(
+                "Expected call format: tag_or_address,entrypoint[,calldata1,...,calldataN]"
+            ));
+        }
+
+        let tag_or_address = ResourceDescriptor::from_string(parts[0])?;
+        let entrypoint = parts[1].to_string();
+        let calldata =
+            if parts.len() > 2 { calldata_decoder::decode_calldata(parts[2])? } else { vec![] };
+
+        Ok(CallArguments { tag_or_address, entrypoint, calldata })
+    }
+}
+
 #[derive(Debug, Args)]
-#[command(about = "Execute a system with the given calldata.")]
+#[command(about = "Execute one or several systems with the given calldata.")]
 pub struct ExecuteArgs {
-    #[arg(
-        help = "The address or the tag (ex: dojo_examples:actions) of the contract to be executed."
-    )]
-    pub tag_or_address: ResourceDescriptor,
+    #[arg(num_args = 1..)]
+    #[arg(help = "A list of calls to execute.\n
+A call is made up of 3 values, separated by a comma (<TAG_OR_ADDRESS>,<ENTRYPOINT>[,<CALLDATA>]):
```

Comment:
May be more natural to have the calldata separated from the two first arguments.

Starkli is doing that: https://github.com/xJonathanLEI/starkli/blob/4c92ac0ee3c8a50289b09760ec535cf27dd64b2c/src/subcommands/invoke.rs#L62

Wondering what could be best to have something easy to use, wdyt on that?